### ISSUE TITLE
Default options for auto-generation not used domain names

### ISSUE THREAD

**Issue by mr-tron** [denizzzka/dianna2#6]
Я считаю, что клиенты должны по-умолчанию в случае отсутствия нагрузки на процессор генерировать неиспользуемые случайный длинные домены. Это создаст на начальных этапах относительно стабильный поток записей. Естественно это должно быть легко отключаемым, но должно быть. Это снимет проблему с тем что за 12 часов записей может не быть.

**Comment by denizzzka** [comment]
Зачем? Отсутствие в течении 12 часов новых записей проблемой не является.

(А вот если записей не будет 9 месяцев то цепочка полностью умрёт и потребуется перезапуск.)

**Comment by mr-tron** [comment]
а отсутствие proof-of-work  в течение недели является. короче на стадии запуска, я считаю это категорически важная мысль.

**Comment by denizzzka** [comment]
нет, ведь можно искуственно отодвинуть время назад при заполнении и в мир выпустить уже готовую цепочку возрастом более двух недель
